cmd/minikube/cmd/config/enable.go

```diff
@@ -76,5 +77,6 @@ var (
 func init() {
 	addonsEnableCmd.Flags().StringVar(&images, "images", "", "Images used by this addon. Separated by commas.")
 	addonsEnableCmd.Flags().StringVar(&registries, "registries", "", "Registries used by this addon. Separated by commas.")
+	addonsEnableCmd.Flags().BoolVar(&gcpauth.Force, "force", false, "If true, will force gcp-auth addon to be enabled on GCE.")
```

Comment:
I am curious if user passes --force using start, will it get picked up by the addon ?
```
minikube start --addons="gcp-auth" --force
```
or does it have to be
```
minikube addons enable gcp-auth --force
```

Reply:
I added a line to the start code make sure that works

Reply:
thanks